aspnetcore/signalr/configuration.md

```diff
@@ -71,6 +71,7 @@ The following table describes options for configuring SignalR hubs:
 | `StreamBufferCapacity` | `10` | The maximum number of items that can be buffered for client upload streams. If this limit is reached, the processing of invocations is blocked until the server processes stream items.|
 | `MaximumReceiveMessageSize` | 32 KB | Maximum size of a single incoming hub message. |
 | `MaximumParallelInvocationsPerClient` | 1 | The maximum number of hub methods that each client can call in parallel before queueing. |
+| `DisableImplicitFromServicesParameters` | false | Hub method arguments will be resolved from DI if possible. |
```

Comment:
To avoid localization by MT ...

```suggestion
| `DisableImplicitFromServicesParameters` | `false` | Hub method arguments will be resolved from DI if possible. |
```